xds/src/main/java/io/grpc/xds/ClusterResolverLoadBalancer.java

```diff
@@ -560,7 +558,7 @@ public void run() {
                 addresses.add(eag);
               }
               PriorityChildConfig priorityChildConfig = generateDnsBasedPriorityChildConfig(
-                  name, edsServiceName, lrsServerName, maxConcurrentRequests, tlsContext,
+                  name, null, lrsServerName, maxConcurrentRequests, tlsContext,
```

Comment:
The second argument of `generateDnsBasedPriorityChildConfig()` can be removed.

Reply:
Good catch! Eliminated it.